README.md

```diff
@@ -39,36 +39,37 @@ This vagrant box aims to make it dead simple to start a hashistack and emulate h
 
 ## Content
 
-1. [Description - what & why](#description---what--why)
-    1. [Submodules](#submodules)
-    2. [Services](#services)
-    3. [Why does this exist?](#why-does-this-exist)
-    4. [Installed stack](#installed-stack)
-    5. [Versions](#versions)
-2. [Install prerequisites](#install-prerequisites)
-    1. [General requirements](#general-requirements)
-        1. [Proxy](#proxy)
-    2. [Linux requirements](#linux-requirements)
-    3. [MacOS requirements](#macos-requirements)
-    4. [Windows requirements](#windows-requirements)
-        1. [GNU Make and Git Bash](#GNU-Make-and-Git-Bash)
-        2. [Installation](#installation)
-3. [Build](#build)
-4. [Configuration](#configuration)
-    1. [Default Configuration](#default-configuration)
-    2. [Override default configuration](#override-default-configuration)
-        1. [ENV variables](#option-1---env-variables)
-        2. [Config files](#option-2---config-files)
-5. [Usage](#usage)
-   1. [Starting a plain default box](#option-1-starting-a-plain-default-box)
+1. [Content](#content)
+2. [Description - what & why](#description---what--why)
+   1. [Submodules](#submodules)
+   2. [Services](#services)
+   3. [Why does this exist?](#why-does-this-exist)
+   4. [Installed stack](#installed-stack)
+   5. [Versions](#versions)
+3. [Install prerequisites](#install-prerequisites)
+   1. [General requirements](#general-requirements)
+      1. [Proxy](#proxy)
+      2. [MacOS](#macos)
```

Comment:
Could you move the content of `MacOS` section under `MacOS requirements` section?

Reply:
Not sure what you mean? That it should be indented under proxy? If so, I've fixed it

Reply:
<img width="585" alt="Screenshot 2020-10-12 at 10 55 52" src="https://user-images.githubusercontent.com/15572799/95726721-911b6580-0c79-11eb-8fa6-364aeffe75cc.png">

Reply:
Actually you right. The content of the section `MacOS` belongs to the proxy context. Lets resolve this conversation